downloader.go

```diff
@@ -85,12 +101,12 @@ func (dl *downloader) Run() bool {
 
 	dl.merger.Close()
 
-	return !dl.failure
+	return nil
 }
 
 func (dl *downloader) extractFileName() {
 	tokens := strings.Split(dl.url, "/")
-	last := tokens[len(tokens) - 1]
+	last := tokens[len(tokens)-1]
```

Comment:
![This change is bad, and you should feel bad](https://i.imgflip.com/2fhwnr.jpg)

Reply:
Not me, `gofmt` is to blame :D